docs/components/ColorTile.js

```diff
@@ -5,34 +5,42 @@ import { Box, Text } from 'gestalt';
 import tokens from 'gestalt-design-tokens/dist/json/variables.json';
 
 type Props = {|
-  name: string,
-  tokenId: string,
-  number: number,
-  textColor: 'darkGray' | 'white',
+  fullTokenName: string,
+  name?: string,
+  number?: number,
+  textColor?: 'darkGray' | 'white',
```

Comment:
Would it be possible to simplify the API by combining `textColor` and `showName`? Something like
```js
textColor: 'darkGray' | 'white' | 'none';
```
where `'none'` is equivalent to `showName={false}`